Wikipedia/Code/ReadingListDetailViewController.swift

```diff
@@ -8,7 +8,7 @@ class ReadingListDetailViewController: ViewController {
     let dataStore: MWKDataStore
     let readingList: ReadingList
     
-    let articlesCollectionViewController: ArticlesCollectionViewController
+    let articlesCollectionViewController: ReadingListEntryCollectionViewController
```

Comment:
given the new name, should `articlesCollectionViewController` be renamed to `readingListEntryCollectionViewController` here and in other places?